### DISCUSSION TITLE
stange user experience regarding attribute lists vss task list content - bug or feature?

### DISCUSSION THREAD

**Discussion by aisbergde** [ransome1/sleek#681] [General]
The user experience regarding gray marked attributes and the task list is very strange for me and I have no idea, ho this could be used
![image](https://github.com/ransome1/sleek/assets/9020187/82e9fac8-6d2f-42e1-82e9-3c0dd1bd424e)
The attributes, which I marked in yellow, are gray in the Attribute list, but they are available in the task list:

if the attributes are used in the task list, why they are grayed out? If they are grayed out, they can't be used to filter.

I also don't understand the logic behind. Why some used attributes are grayed out, and others are shown?
for example, "Bookmark" is not grayed out, while "Irgendwann" is grayed out. But both are used: 
![image](https://github.com/ransome1/sleek/assets/9020187/625e2097-4600-4fcc-8de8-acd04121eb7c)

Is it a bug or a feature?

**Comment by ransome1** [comment]
@aisbergde what version are you on? If it is one of the latest pre-releases, it is possible to be a bug, since there is work ongoing on the attribibutes.

Does this happen in v2.0.11 too?

**Comment by aisbergde** [comment]
I am using v2.0.11
And I have seen, the issue is related to the setting `hide / show hidden tasks`.
If I hide them, the result are correct
![image](https://github.com/ransome1/sleek/assets/9020187/e04b590a-b9e2-4f64-96a2-84a56416f60f)
![image](https://github.com/ransome1/sleek/assets/9020187/2b2984c3-6e22-4a68-b14f-b9afe6e9b277)

and if not hidden:
![image](https://github.com/ransome1/sleek/assets/9020187/ac3d25b2-dfbf-430e-ac99-7a15bf129cc1)

The content of the hidden tasks are
```
2024-03-10 h:1 @Bookmark @HowTo @Irgendwann @Lesen @Mission @WartenAuf +Anna +Felicia +Germo +Rebecca
2024-03-12 h:1 Manage Your Now @@CriticalNow @@OpportunityNow @@OverTheHorizon @@!SOC

```

**Reply by ransome1** [reply]
@aisbergde this is how the hidden setting is designed and described in the wiki.

**Reply by aisbergde** [reply]
but the result is wrong. There are visible tasks containing entries, but the entries are shown as disabled. They should not be disabled, if there are any not hidden tasks, containing these entries. It looks like the setting will mark all entries as disabled, which are uses in any hidden task. And this is the issue. These entries should be hidden only, when they are used only in hidden tasks.

**Reply by ransome1** [reply]
can you please provide a full working example, so that I can reproduce this?

**Comment by aisbergde** [comment]
similar issue, when "completed tasks" are hidden. Maybe other filters also leads to this issue.

**Reply by ransome1** [reply]
@aisbergde this was actually an issue, at least for "completed tasks" as well as the "... date in the future" setting. It should be resolved in the latest pre-release: https://github.com/ransome1/sleek/releases/tag/v2.0.13-rc.1

**Reply by aisbergde** [reply]
solved in v2.0.13-rc.1
I can't reproduce the issue in this version.
